src/wallet.js

```diff
@@ -876,6 +876,51 @@ const wallet = {
     return data;
   },
 
+  /**
+   * Filter an utxo based on the specified options
+   *
+   * @param {object} output Transaction output to be filtered/validated
+   * @param {object} utxoDetails utxos and meta information
+   * @param {object} options Consolidation options shared with getUtxos and consolidateUtxos (https://github.com/HathorNetwork/hathor-wallet-headless/issues/44)
```

Comment:
We should document here the options here.

Reply:
Done